### PR TITLE
Build and subscribe to one query id via CLI

### PR DESCRIPTION
**Pull request by tallywiesenberg**
As requested by @themandalore

### CODE REVIEW

File: tests/test_data.py [hunk 1]
```diff
@@ -138,16 +138,17 @@ async def test_parse_new_report_event():
...
     cfg.main.chain_id = 5
-    tx_hash = "0xf7fb66b0c3961692cd9658ce4a8c5e73ba8fbc954676d417e815456337604797"
+    tx_hash = "0xb3e4fb09e3dcc7abc0f30181c5dc9ba7253d694de17d88c37c2bd4069f0eebdc"
 
     for endpoint in cfg.endpoints.find(chain_id=5):
         cfg.endpoints.endpoints.remove(endpoint)
 
-    endpoint = RPCEndpoint(5, "Goerli", "Infura", os.getenv("NODE_URL"), "etherscan.io")
+    endpoint = RPCEndpoint(
+        5, "Goerli", "Infura", "https://goerli.infura.io/v3/db7ce830b1224efe93ae3240f7aaa764", "etherscan.io"
```
Comment (oraclown) [inline tests/test_data.py:147]: Thy API key?

File: src/tellor_disputables/cli.py [hunk 2]
```diff
@@ -29,15 +30,22 @@ def print_title_info() -> None:
 
 
 @click.command()
-@click.option("-a", "--all-values", is_flag=True, show_default=True)
+@click.option("-a", "--all-values", is_flag=True, show_default=True, help="if set, get alerts for all values")
 @click.option("-w", "--wait", help="how long to wait between checks", type=int)
+@click.option("-f", "--filter", help="build a feed and get alerts for that feed only", is_flag=True)
+@click.option(
```
Comment (oraclown) [inline src/tellor_disputables/cli.py:36]: maybe add a default for -c

File: src/tellor_disputables/cli.py [hunk 3]
```diff
@@ -29,15 +30,22 @@ def print_title_info() -> None:
 
 
 @click.command()
-@click.option("-a", "--all-values", is_flag=True, show_default=True)
+@click.option("-a", "--all-values", is_flag=True, show_default=True, help="if set, get alerts for all values")
 @click.option("-w", "--wait", help="how long to wait between checks", type=int)
+@click.option("-f", "--filter", help="build a feed and get alerts for that feed only", is_flag=True)
```
Comment (oraclown) [inline src/tellor_disputables/cli.py:35]: confusing use of "feed" when you're  filtering based on query id

File: src/tellor_disputables/cli.py [hunk 4]
```diff
@@ -29,15 +30,22 @@ def print_title_info() -> None:
 
 
 @click.command()
-@click.option("-a", "--all-values", is_flag=True, show_default=True)
+@click.option("-a", "--all-values", is_flag=True, show_default=True, help="if set, get alerts for all values")
```
Comment (oraclown) [inline src/tellor_disputables/cli.py:33]: Is the default to check all query ids?

File: pyproject.toml [hunk 5]
```diff
@@ -16,6 +16,7 @@ pytest-asyncio = "^0.19.0"
 telliot-core = {git = "https://github.com/tellor-io/telliot-core.git", rev = "main"}
 telliot-feeds = {git = "https://github.com/tellor-io/telliot-feeds.git", rev = "main"}
 click = "^8.1.3"
+pydantic = "^1.10.2"
```
Comment (oraclown) [inline pyproject.toml:19]: This dep is getting removed from telliot-core/feeds in next release just fyi

**Review by oraclown — CHANGES_REQUESTED** [review]
Thanks @tallywiesenberg ! Nice test. I left some suggestions

File: src/tellor_disputables/cli.py (hunk 4 above)
Comment (tallywiesenberg) [reply src/tellor_disputables/cli.py:33]: Good catch, adding explicit default

File: tests/test_data.py (hunk 1 above)
Comment (tallywiesenberg) [reply tests/test_data.py:147]: Yes, test wasn't loading api key though so I left it. We could debug it together

File: src/tellor_disputables/cli.py (hunk 3 above)
Comment (tallywiesenberg) [reply src/tellor_disputables/cli.py:35]: good catch

File: pyproject.toml (hunk 5 above)
Comment (tallywiesenberg) [reply pyproject.toml:19]: thanks

**Review by oraclown — APPROVED** [review]
Nice updates, sire

**Pull request merged**